cost/aws/reserved_instances/recommendations/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v2.18
+
+- Renamed `Reserved Instance` policy_set from info block to `Reserved Instances`.
+- Added `resourceType` incident field.
+- Added `term` incident field.
+- Updated `platform` incident field to avoid empty values.
+- Renamed `accountScope` incident field to `scope`.
```

Comment:
Similar to above, these should be worded with the user in mind. Since users will generally not know or care about the code names of incident fields in the policy, I would consolidate these into a single entry that says something like this:

- Changed internal names of several incident fields to ensure that they are properly scraped for dashboards.

Reply:
Added that line for all three policies!